src/components/va-accordion/fixtures/va-accordion.fixture.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just as an FYI, these stories won't appear on design.va.gov/storybook. If you want them to, you'll have to make another PR based off of `master` and add the stories there :upside_down_face: